packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpPageView.tsx

```diff
@@ -53,7 +53,7 @@ const CdpPageView = (): JSX.Element => {
    * Determines if the page view events should be turned off.
    * IMPORTANT: You should implement based on your cookie consent management solution of choice.
    * You may also wish to disable in development mode (process.env.NODE_ENV === 'development').
```

Comment:
This comment is a duplicate of the updated comment below
Just use: _By default it is disabled in development mode_